package.json

```diff
@@ -148,7 +147,7 @@
     "raphael": "^2.2.8",
     "raven-js": "^3.27.1",
     "request": "^2.51.0",
-    "uri.js": "^0.1.3",
+    "urijs": "^1.19.1",
```

Comment:
No breaking changes ? In case they don't respect semantic versioning

Reply:
Nothing that impact the application codebase.